src/commands/release.ts

```diff
@@ -0,0 +1,38 @@
+import { flags as oclifFlags } from '@oclif/command'
+
+import { CustomCommand } from '../oclif/CustomCommand'
+import appsRelease from '../modules/release'
+
+export default class Release extends CustomCommand {
+  static description =
+    'Bump app version, commit and push to remote. Only for git users. The first option can also be a specific valid semver version'
+
+  static examples = [
+    'vtex release',
+    'vtex release patch',
+    'vtex release patch beta',
+    'vtex release minor stable',
+    'vtex release pre',
```

Comment:
Maybe add `vtex release prerelease` too?

Reply:
I don't think it is truly necessary, since `vtex release pre` is a shorter form of `vtex release prerelease` and it could be too verbose if we add too many examples.